Web.md

```diff
@@ -124,6 +124,86 @@ MIME type mismatch or `opaque` response types
 [reject](http://tc39.github.io/ecma262/#sec-rejectpromise) the Promise with a
 `WebAssembly.CompileError`.
 
+### Structured serialization of a `WebAssembly.Memory`
+
+A `WebAssembly.Memory` is under some circumstances a
```

Comment:
Remove "under some circumstances", here and below. It's always a serializable object; sometimes serialization throws.